core/tests/puppeteer-acceptance-tests/user-utilities/blog-post-admin-utils.js

```diff
@@ -0,0 +1,461 @@
+        if (tagList[i].value === tagName) {
+          throw new Error('Tag ' + tagName + ' already exists in tag list!');
+        }
+      }
+    }, tagName);
+    showMessage('Tag with name ' + tagName + ' does not exist in tag list!');
+  }
+
+  /**
+   * This function adds a new tag in the blog taglist.
+   * @param {string} tagName - The name of the tag.
+   */
+  async addNewBlogTag(tagName) {
+    await this.clickOn(LABEL_FOR_ADD_ELEMENT_BUTTON);
+    await this.page.waitForTimeout(100);
+    await this.page.evaluate((tagName) => {
+      const tagList = document.getElementsByClassName('form-control');
+      tagList[tagList.length - 1].value = tagName;
+    }, tagName);
+    await this.clickOn('Save');
+    showMessage('Tag ' + tagName + ' added in tag list successfully!');
+  }
+
+  /**
+   * This function checks tag exists in the blog taglists.
+   * @param {string} tagName - The name of the tag.
+   */
+  async expectTagToExistInBlogTags(tagName) {
+    await this.page.evaluate(async(tagName) => {
+      const tagList = document.getElementsByClassName('form-control');
+      for (let i = 0; i < tagList.length; i++) {
+        if (tagList[i].value === tagName) {
+          return;
+        }
+      }
+      throw new Error('Tag ' + tagName + ' does not exist in tag list!');
+    }, tagName);
+    showMessage('Tag with name ' + tagName + ' exists in tag list!');
+  }
+
+  /**
+   * This function changes the blog tags limit.
+   * @param {number} limit - The limit of the blog tags.
+   */
+  async setMaximumTagLimitTo(limit) {
+    // These steps are for deleting the existing value in the input field.
+    const tagInputField = await this.page.$(maximumTagLimitInput);
+    await tagInputField.click({ clickCount: 3 });
+    await this.page.keyboard.press('Backspace');
+
+    await this.type(maximumTagLimitInput, limit);
+    await this.clickOn('Save');
+
+    showMessage('Successfully updated the tag limit to ' + limit + '!');
+  }
+
+  /**
+   * This function checks if the tag limit is not equal to.
+   * @param {number} limit - The limit of the blog tags.
+   */
+  async expectMaximumTagLimitNotToBe(limit) {
+    await this.page.evaluate(async(limit) => {
+      const tagLimit = document.getElementById('mat-input-0').value;
+      if (tagLimit.value === limit) {
+        throw new Error('Maximum tag limit is already ' + limit + '!');
+      }
+    }, limit);
+    showMessage('Maximum tag limit is not ' + limit + '!');
+  }
+
+  /**
+   * This function checks if the tag limit is equal to.
+   * @param {number} limit - The limit of the blog tags.
+   */
+  async expectMaximumTagLimitToBe(limit) {
+    await this.page.evaluate(async(limit) => {
+      const tagLimit = document.getElementById('mat-input-0').value;
+      if (tagLimit.value !== limit) {
+        throw new Error('Maximum tag limit is not ' + limit + '!');
+      }
+    });
+    showMessage('Maximum tag limit changed to ' + limit + '!');
```

Comment:
changed to --> is currently

(you are just reporting the current state)

Reply:
done